cypress/support/commands.js

```diff
@@ -126,6 +134,25 @@ const createProposal = (proposalTitle = '', proposalAbstract = '') => {
   cy.contains('Save and continue').click();
 };
 
+const dragElement = (element, moveArgs) => {
```

Comment:
Added new command so anytime someone needs to move element somewhere can use it. 🐉